api/core/domain/models/_mistral.py

```diff
@@ -234,7 +234,29 @@ def mistral_models() -> dict[Model, ModelData | LatestModel | DeprecatedModel]:
             provider_for_pricing=Provider.MISTRAL_AI,
             icon_url="https://workflowai.blob.core.windows.net/workflowai-public/mistral.svg",
             release_date=date(2024, 7, 24),
-            quality_index=481,  # MMLU=63.47, GPQA=38.35
+            quality_data=QualityData(mmlu=63.47, gpqa=38.35),
+            provider_name=DisplayedProvider.MISTRAL_AI.value,
+            supports_tool_calling=True,
+        ),
+        Model.MISTRAL_MEDIUM_2505: ModelData(
+            display_name="Mistral Medium (25-05)",
+            supports_json_mode=True,
+            supports_input_image=True,
+            supports_input_pdf=True,
+            supports_input_audio=False,
+            max_tokens_data=MaxTokensData(
+                max_tokens=262144,
```

Comment:
<img width="818" alt="Screenshot 2025-05-08 at 19 04 46" src="https://github.com/user-attachments/assets/eab75a2d-e3b3-4734-9704-a5663daa1f12" />
are you sure it's not 128k ?

Reply:
You are correct ! fixed